examples/googleform/selectForm.js

```diff
@@ -0,0 +1,36 @@
+'use strict';
+
+const Enquirer = require('enquirer');
+const GoogleFormPrompt = require('..');
```

Comment:
This should come from `prompt-google-form`.

```suggestion
const GoogleFormPrompt = require('prompt-google-form');
```